code/hud/hudconfig.h

```diff
@@ -92,6 +95,45 @@ struct HC_gauge_region
 	HC_gauge_region(const char *name, int x1, int y1, int h, int iff, int cp, int b, int nf, int cl) : filename(name), x(x1), y(y1), hotspot(h), use_iff(iff), can_popup(cp), bitmap(b), nframes(nf), color(cl){}
 };
 
+class BoundingBox {
+  public:
+	int x1, x2, y1, y2;
+
+	// Default constructor (initializes to invalid state)
+	BoundingBox() : x1(-1), x2(-1), y1(-1), y2(-1) {}
+
+	// Constructor
+	BoundingBox(int nx1, int nx2, int ny1, int ny2) : x1(nx1), x2(nx2), y1(ny1), y2(ny2) {}
```

Comment:
consider making these constructors `constexpr`.
It's not super critical here, but we should start adding it where sensible and where things (especially default values) are constructed from literals, just to save us a little bit of performance that is spent on constructing these things.